content/Rust-1.91.0.md

```diff
@@ -0,0 +1,170 @@
++++
+path = "2025/10/30/Rust-1.91.0"
+title = "Announcing Rust 1.91.0"
+authors = ["The Rust Release Team"]
+aliases = ["releases/1.91.0"]
+
+[extra]
+release = true
++++
+
+The Rust team is happy to announce a new version of Rust, 1.91.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via `rustup`, you can get 1.91.0 with:
+
+```console
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`](https://www.rust-lang.org/install.html) from the appropriate page on our website, and check out the [detailed release notes for 1.91.0](https://doc.rust-lang.org/stable/releases.html#version-1910-2025-10-30).
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you might come across!
+
+## What's in 1.91.0 stable
+
+### `aarch64-pc-windows-msvc` is now a Tier 1 platform
+
+The Rust compiler supports [a wide variety of targets][platform-support], but
+the Rust Team can't provide the same level of support for all of them. To
+clearly mark how supported each target is, we use a tiering system:
+
+* Tier 3 targets are technically supported by the compiler, but we don't check
+  whether their code build or passes the tests, and we don't provide any
+  prebuilt binaries as part of our releases.
+* Tier 2 targets are guaranteed to build and we provide prebuilt binaries, but
+  we don't execute the test suite on those platforms: the produced binaries
+  might not work or might have bugs.
+* Tier 1 targets provide the highest support guarantee, and we run the full
+  suite on those platforms for every change merged in the compiler. Prebuilt
+  binaries are also available.
+
+Rust 1.91.0 promotes the `aarch64-pc-windows-msvc` target to Tier 1 support,
+bringing our highest guarantees to users of 64-bit ARM systems running Windows.
+
+### Add lint against dangling raw pointers from local variables
+
+While Rust's borrow checking prevents dangling references from being returned, it doesn't
+track raw pointers. With this release, we are adding a warn-by-default lint on raw
+pointers to local variables being returned from functions. For example, code like this:
+
+```rust
+fn f() -> *const u8 {
+    let x = 0;
+    &x
```

Comment:
```suggestion
    &raw const x
```

Obviously this coerces anyway, but it seems maybe a bit better to not leave any question in the example about whether this implicit coercion is part of what's triggering this lint.  Also, it's good to take the opportunity to familiarize people with the raw borrow syntax.

Reply:
Either way really.  Pros and cons.